src/main/java/com/pm/backend/model/v1/question/QuestionModel.java

```diff
@@ -0,0 +1,28 @@
+package com.pm.backend.model.v1.question;
+
+
+import io.swagger.annotations.ApiModelProperty;
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.util.List;
+
+@Data
+
```

Comment:
Hmm i don't see the getters/setters lombok annotation here

Reply:
@Data includes @Setter and @Getter